nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-administration/src/test/resources/nifi-administration-context.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Where is this used?

Reply:
I must have set this up for some tests that I abandoned. I'll remove this from the PR.